kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/BigQueryHelper.java

```diff
@@ -29,38 +29,49 @@
 import java.io.FileInputStream;
 import java.io.IOException;
 import java.io.InputStream;
+import java.io.ByteArrayInputStream;
+import java.nio.charset.StandardCharsets;
 
 /**
  * Convenience class for creating a default {@link com.google.cloud.bigquery.BigQuery} instance,
  * with or without login credentials.
  */
 public class BigQueryHelper {
   private static final Logger logger = LoggerFactory.getLogger(BigQueryHelper.class);
+  private static String keyFileType;
 
   /**
    * Returns a default {@link BigQuery} instance for the specified project with credentials provided
    * in the specified file, which can then be used for creating, updating, and inserting into tables
    * from specific datasets.
    *
-   * @param projectName The name of the BigQuery project to work with
-   * @param keyFilename The name of a file containing a JSON key that can be used to provide
-   *                    credentials to BigQuery, or null if no authentication should be performed.
```

Comment:
For some reason, these two params were causing an error during build since they weren't found in the class. I'm not sure how it passed the build before.